content/en/blog/_posts/2018-04-11-migrating-the-kubernetes-blog.md

```diff
@@ -23,7 +23,7 @@ We hope that making blog submissions more accessible will encourage greater comm
 You can submit a blog post for consideration one of two ways:
 
 * Submit a Google Doc through the [blog submission form](https://docs.google.com/forms/d/e/1FAIpQLSch_phFYMTYlrTDuYziURP6nLMijoXx_f7sLABEU5gWBtxJHQ/viewform)
-* Open a pull request against the [website repository](https://github.com/kubernetes/website/tree/master/blog) as described [here](/docs/home/contribute/create-pull-request/)
+* Open a pull request against the [website repository](https://github.com/kubernetes/website/tree/master/content/en/blog) as described [here](/docs/home/contribute/create-pull-request/)
```

Comment:
Sorry to keep nitpicking on this PR, but it looks to me as though this link should go one directory deeper for clear guidance: so it would be:
https://github.com/kubernetes/website/tree/master/content/en/blog/_posts

Reply:
No worries - happy to improve this further... thanks